generators/newapp/templates/actions/app.go.tmpl

```diff
@@ -34,6 +34,9 @@ var T *i18n.Translator
 // should be defined. This is the nerve center of your
 // application.
 func App() *buffalo.App {
+  //Loading .env file values
+  envy.Load()
```

Comment:
Move this into `buffalo.New` that way it'll just work for every app, and they don't need the extra line. Check to make sure that moving this doesn't cause any problems for older apps, as well as apps that don't have a `.env`.